### ISSUE TITLE
Area between curves throws VisibleDeprecationWarning: Creating an ndarray from ragged nested sequences

### ISSUE THREAD

**Issue by stefanmeili** [holoviz/holoviews#4779]
### Problem Description
A recent update to numpy deprecates creation of ndarrays from ragged nested sequences. This affects Holoviews Area plots between curves.

As a result, my output is overwhelmed by pink warning messages.

### Versions
Name                    Version                   Build 
holoviews              1.14.0             pyhd3eb1b0_0  
numpy                   1.19.2           py38h54aff64_0  
numpy-base          1.19.2           py38hfa32c7d_0  
bokeh                     2.2.3                    py38_0  
ipython                   7.19.0           py38hb070fc8_0  
python                    3.8.5                h7579374_1  

### Example
Example pulled from https://holoviews.org/reference/elements/bokeh/Area.html
```python
import numpy as np

import holoviews as hv
hv.extension('bokeh')

X  = np.linspace(0,3,200)
Y = X**2 + 3
Y2 = np.exp(X) + 2
Y3 = np.cos(X)
hv.Area((X, Y, Y2), vdims=['y', 'y2']) * hv.Area((X, Y, Y3), vdims=['y', 'y3'])
```

### Backtrace
/miniconda3/envs/ML/lib/python3.8/site-packages/holoviews/core/util.py:980: VisibleDeprecationWarning: Creating an ndarray from ragged nested sequences (which is a list-or-tuple of lists-or-tuples-or ndarrays with different lengths or shapes) is deprecated. If you meant to do this, you must specify 'dtype=object' when creating the ndarray
  arr = np.array(values)

### Screenshot
![image](https://user-images.githubusercontent.com/16913760/103800597-73079c00-5001-11eb-9418-3f83bcf90afc.png)

**Comment by stefanmeili** [comment]
A workaround:
 ```
np.warnings.filterwarnings('ignore', category=np.VisibleDeprecationWarning)
```

**Comment by jlstevens** [comment]
Thanks for reporting this!

We'll make sure to silence those warning messages with the necessary fix.

**Comment by jlstevens** [comment]
I believe this is fixed in #4812. CLosing.
